src/core/public/fatal_errors/fatal_errors_service.tsx

```diff
@@ -56,7 +78,7 @@ export class FatalErrorsService {
       });
   }
 
-  public add = (error: Error | string, source?: string) => {
+  public add: FatalErrorsSetup['add'] = (error, source?) => {
```

Comment:
private. no-one should call it on an instance

Reply:
:+1: